test/models/site_test.rb

```diff
@@ -61,4 +61,35 @@ class SiteTest < ActiveSupport::TestCase
       assert_equal(["はすでに存在します"], site.errors[:fqdn])
     end
   end
+
+  sub_test_case "category_title_format" do
+    test "null" do
```

Comment:
ref: https://github.com/bm-sms/daimon-news/pull/458#discussion_r59831878
also this.